applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/server_store.h

```diff
@@ -0,0 +1,413 @@
+ * @param[in] addr  Pointer to the address to find the server for.
+ *
+ * @note	srv_store_lock() must be called before accessing this function.
+ *
+ * @retval	true if the server exists, false otherwise.
+ */
+bool srv_store_server_exists(bt_addr_le_t const *const addr);
+
+/**
+ * @brief	Get a server from the dst address in the conn pointer.
+ *
+ * @note	srv_store_lock() must be called before accessing this function.
+ *
+ * @param[in] conn  Pointer to the connection to find the server for.
+ * @param[out] server  Pointer to the server structure to fill. NULL if not found.
+ *
+ * @retval	0 Success.
+ * @retval	-ENOENT Server not found in storage.
+ * @retval	-ENOTCONN Server conn is NULL.
+ */
+int srv_store_from_conn_get(struct bt_conn const *const conn, struct server_store **server);
+
+/**
+ * @brief	Get the number of stored servers.
+ *
+ * @note	srv_store_lock() must be called before accessing this function.
+ * @note	Do not iterate through the servers, as they may not be stored consecutively in
+ * memory. Use the srv_store_foreach_func_t function.
+ *
+ *
+ * @retval	number of servers
+ * @retval	-EINVAL Non-consecutive server storage detected.
+ */
+int srv_store_num_get(void);
+
+/**
+ * @brief	Add a server to the storage based on conn.
+ *
+ * @param[in] conn  Pointer to the connection associated with the server to add, based on address.
+ *
+ * @note	This function should not be used if the peer uses a resolvable address
+ * which has not yet been resolved, or any other changing address type.
+ * srv_store_lock() must be called before accessing this function.
+ *
+ * @retval	0 Success.
+ * @retval	-EALREADY The server already exists.
+ * @retval	-ENOMEM  Out of memory.
+ * @retval	Other negative.	Errors from underlying functions.
+ */
+int srv_store_add_by_conn(struct bt_conn *conn);
+
+/**
+ * @brief	Add a server to the storage based on address.
+ *
+ * @param[in] addr  Pointer to the address associated with the server to add.
+ *
+ * @note	This function can be used when getting addresses from bonding information.
+ * At this point, there will only be an address associated with the server, no conn
+ * pointer, and so on. Hence, you must call srv_store_conn_update() when a connection is
+ * established.
+ *
+ * @note srv_store_lock() must be called before accessing this function.
+ *
+ * @retval	0 Success.
+ * @retval	-EALREADY The server already exists.
+ * @retval	-ENOMEM  Out of memory.
+ * @retval	Other negative.	Errors from underlying functions.
+ */
+int srv_store_add_by_addr(const bt_addr_le_t *addr);
+
+/**
+ * @brief	Update the conn pointer of an existing server, based on address.
+ *
+ * @param[in] conn  Pointer to the connection to write.
+ * @note	This function can be used when an earlier pairing has been done to this server.
+ *
+ * @note srv_store_lock() must be called before accessing this function.
+ * bonded device added by srv_store_add_by_addr(). srv_store_lock() must be called before accessing
+ * this function.
```

Comment:
There's clearly something wrong here:
```suggestion
 * @note srv_store_lock() must be called before accessing this function.
 * bonded device added by srv_store_add_by_addr().
```
What is missing in the second line?

Also, move the entire note _before_ param.